projects/storefrontapp-e2e-cypress/cypress/e2e/accessibility/product-details-page.a11y-e2e.cy.ts

```diff
@@ -0,0 +1,46 @@
+/*
+ * SPDX-FileCopyrightText: 2025 SAP Spartacus team <spartacus-team@sap.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as siteContextSelector from '../../helpers/site-context-selector';
+
+/**
+ * This test checks accessibility concerns on the PDP page using Access Continuum
+ */
+describe('Product Details Page Accessibility', () => {
+  beforeEach(() => {
+    cy.a11yContinuumSetup();
+    cy.visit(siteContextSelector.PRODUCT_PATH_2);
+  });
+
+  it('PDP with Product Details', () => {
+    cy.get('cx-page-slot[position="Tabs"] .tab button:nth-child(1)').click();
+    cy.get(
+      'cx-tab-panel [role="tabpanel"].active cx-product-details-tab .container'
+    );
+    cy.get('main').a11yRunContinuumTest();
+  });
+
+  it('PDP with Reviews', () => {
+    cy.get('cx-page-slot[position="Tabs"] .tab button:nth-child(3)').click();
+    cy.get(
+      'cx-tab-panel [role="tabpanel"].active cx-product-reviews .container .header'
+    ).a11yRunContinuumTest();
+  });
+
+  it('PDP with Spec', () => {
+    cy.get('cx-page-slot[position="Tabs"] .tab button:nth-child(2)').click();
+    cy.get(
+      'cx-tab-panel [role="tabpanel"].active cx-product-attributes .container'
+    ).a11yRunContinuumTest();
+  });
+
+  it('PDP with Shipping', () => {
+    cy.get('cx-page-slot[position="Tabs"] .tab button:nth-child(4)').click();
+    cy.get(
+      'cx-tab-panel [role="tabpanel"].active cx-paragraph'
```

Comment:
```suggestion
      '[position="Tabs"] cx-tab-panel .active''
```

We need to run this on the whole panel not just a single element.

Reply:
Applied!